NDcPP_v4_0-draft.adoc

```diff
@@ -1988,20 +1988,10 @@ As indicated in the introduction to this cPP, the baseline requirements (those t
 * Configuration of a new time server
 * Removal of configured time server
 |Identity if new/removed time server
-|FDP_CER_EXT.1 |Certificate generation |Identification of certificate generated
 |FIA_AFL.1 |Unsuccessful login attempts limit is met or exceeded. |Origin of the attempt (e.g., IP address).
 |FIA_PMG_EXT.1 |None. |None.
 |FIA_TSM_EXT.1 |Any addition, replacement, or removal of trust anchors in the TOE's trust store |Identification of trust anchor certificate
```

Comment:
The audit event for FIA_TSM_EXT.1should be removed at that's from the FP for X.509.